azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -33,6 +33,8 @@
   public static final String CONCURRENT_OPTION_SKIP = "skip";
   public static final String CONCURRENT_OPTION_PIPELINE = "pipeline";
   public static final String CONCURRENT_OPTION_IGNORE = "ignore";
+  public static final String FLOW_PRIORITY = "flowPriority";
+  public static final int DEFAULT_FLOW_PRIORITY = 5;
```

Comment:
From user's perspective it will be covered in documentation and from code perspective examples and details are defined in ExecutableFlowPriorityComparator and ExecutableFlowPriorityComparatorTest